afteraction-share.html

```diff
@@ -409,6 +406,22 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 							}
 						}
 					}
+
+					// Set Whatsapp Share URL based on response from API
+					var getShareURL = '/share/link?type=ot&page_name={{page.name}}&short=1';
+					$.ajax({
+						method: 'GET',
+						url: getShareURL,
+					}).done(function(shareURL) {
+						var currWhatsappShareEls = $('a.whatsapp-share');
+						if (currWhatsappShareEls.length > 0) {
```

Comment:
Is this checking for multiple whatsapp shares on a page? What's the case where we would have multiple, if so?